src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx

```diff
@@ -49,20 +61,22 @@ export const AssetSelectorScreen = ({assetTokens, onSelect, onSelectAll}: Props)
       </View>
 
       <FlatList
-        data={sortedAssetTokens}
-        renderItem={({item: assetToken}) => (
+        data={sortedBalance}
+        renderItem={({item: [tokenId, quantity]}: {item: [TokenId, Quantity]}) => (
           <Boundary>
             <AssetSelectorItem
-              key={assetToken.identifier}
-              assetToken={assetToken}
+              wallet={wallet}
+              key={tokenId}
```

Comment:
~~key~~
already has keyExtractor

Reply:
only necessary when using `Array.prototype.map`
and even then, it should be on the wrapper component

Reply:
done